core-bundle/src/Controller/InsertTagsController.php

```diff
@@ -26,20 +25,16 @@
  */
 class InsertTagsController
 {
-    private ContaoFramework $framework;
+    private InsertTagParser $insertTagParser;
 
-    public function __construct(ContaoFramework $framework)
+    public function __construct(InsertTagParser $insertTagParser)
     {
-        $this->framework = $framework;
+        $this->insertTagParser = $insertTagParser;
     }
 
     public function renderAction(Request $request, string $insertTag): Response
     {
-        $this->framework->initialize();
-
-        $it = $this->framework->createInstance(InsertTags::class);
-
-        $response = new Response($it->replace($insertTag, false));
+        $response = new Response($this->insertTagParser->replaceInline($insertTag));
```

Comment:
@ausi Just double checking here: If `$blnCache` is `false`, we have to use the `replaceInline()` method, don't we?

Reply:
✅